example_files/roary_mini_example.csv

```diff
@@ -1,5 +1,5 @@
 Gene,Non-unique Gene name,Annotation,No. isolates,No. sequences,Avg sequences per isolate,Genome Fragment,Order within Fragment,Accessory Fragment,Accessory Order with Fragment,QC,Min group size nuc,Max group size nuc,Avg group size nuc,s1,s2,s3,s4,s5
-a,,,,,,,,,,,,,,x,,x,x,
+a,abc,hypothetical protein,,,,,,,,,,,,x,,x,x,
```

Comment:
can we use something else than "abc" for the gene name, as this might be confusing with the genes a, b, c. How about "gene_a"?

Reply:
We can also change this later. Will merge now.

Reply:
Sorry, did not see this earlier! Done